CAM/RenderScriptCams.md

```diff
@@ -27,3 +27,4 @@ NativeDB Added Parameter 6: Any p5
 * **p3**: 
 * **p4**: 
 
+Note: if you are having issues with weird camera ease start positions, try setting p3 and p4 to `true`.
```

Comment:
This line needs to be moved up to above the Parameters section to work correctly in the description. Also please remove the code blocks from the description.